scripts/nb-tester/test-notebook.py

```diff
@@ -158,7 +154,7 @@ def _execute_notebook(filepath: Path, args: argparse.Namespace) -> nbformat.Note
 
     processor = nbconvert.preprocessors.ExecutePreprocessor(
         # If submitting jobs, we want to wait forever (-1 means no timeout)
-        timeout=-1 if submit_jobs else 100,
+        timeout=-1 if submit_jobs else 300,
```

Comment:
Some cells take a bit longer to execute, possibly because they use multithreading